core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftScanner.java

```diff
@@ -311,14 +311,12 @@ else if (log.isTraceEnabled())
 
             lastError = error;
             sleepMillis = pause(sleepMillis, maxSleepTime);
-          } finally {
-            locateSpan.stop();
           }
         }
 
-        Span scanLocation = Trace.start("scan:location");
-        scanLocation.data("tserver", loc.tablet_location);
-        try {
+        try (TraceScope scanLocation = Trace.startSpan("scan:location")) {
+          scanLocation.getSpan().addKVAnnotation("tserver".getBytes(UTF_8),
```

Comment:
The new API no longer takes strings?

Reply:
The old API never took strings... it took `byte[]`. We had a utility class to do this, but it was confusing, so I inline'd it. HTrace 3.2 and later API actually changed it to String, so these will get simpler when I get that far.